components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -441,7 +441,8 @@ public static OAuthConsumerAppDTO buildConsumerAppDTO(OAuthAppDO appDO) {
         dto.setApplicationAccessTokenExpiryTime(appDO.getApplicationAccessTokenExpiryTime());
         dto.setRefreshTokenExpiryTime(appDO.getRefreshTokenExpiryTime());
         dto.setIdTokenExpiryTime(appDO.getIdTokenExpiryTime());
-        dto.setAudiences(appDO.getAudiences());
+        dto.setIdTokenAudiences(appDO.getIdTokenAudiences());
+        dto.setAccessTokenAudiences(appDO.getAccessTokenAudiences());
```

Comment:
need to preserve the backward compatibility

Reply:
resolved with 08954d00bccda5645fa04905eca48d4cc7ae5f01